Kotlin/sort/QuickSort/src/QuickSort.kt

```diff
@@ -0,0 +1,73 @@
+//Kotlin program to implement QuickSort using recursive approach
```

Comment:
Few changes: 

- Add Multi-Line Comments 
- Add a beginner-friendly description at the top